content/README.md

```diff
@@ -0,0 +1,15 @@
+# Entando Hub Content Bundle
+
+## Overview
+This bundle requires `entando-hub-application-bundle`. <br>
+This is the content bundle for the Entando Hub. It contains the "non-code" parts of the Entando Hub including a basic page setup.
+
+
+## Deployment and Installation
+This project is ent-enabled so you can use the Entando CLI (ent) to perform the full build and deployment sequence.
+
```

Comment:
We should add something about the script [set-bundle-id.sh](https://github.com/entando-samples/entando-hub/blob/develop/content/set-bundle-id.sh) that must be used to connect externally referenced bundles (in this case the application bundle). At the beginning of the script are present some instructions.

Reply:
@firegloves The set-bundle-id.sh seems odd to include in the Hub. I wonder if something like this would be more useful within the CLI since it's not specific to the Hub.

Reply:
@nshaw you are right, we decided to create it to allow every one of us to work efficiently on the hub. By migrating to bundle v5 we introduced some hard dependencies between the 2 bundles (application and content) specified by their bundle ids. While developing we are probably publishing on our personal repos, so we need a quick and easy way to update bundle ids to point to the valid bundles. Of course, we are planning to update the CLI, but we were reasoning about a clean way to introduce dependencies between bundles in that case. This is a temporary solution that we will remove also from docs when we will address the main issue

Reply:
@firegloves sounds good to me. It's making me think the bundle id approach is too brittle in the current design. I understand the reasons it was added but we might need a conversation around how to make this easier since this will definitely reduce developer joy. We're effectively embedding a namespace (bundle id) into the names of numerous objects and it feels like duplication (or a code smell).